pkgs/development/tools/steamos-devkit/default.nix

```diff
@@ -0,0 +1,117 @@
+{ lib
+, fetchFromGitHub
+, fetchFromGitLab
+, writeScript
+, python3
+, makeDesktopItem
+, pkg-config
+, SDL2
+}:
+let
+  # steamos-devkit requires a build of the unreleased pyimgui 2.0 branch, move to pythonPackages when 2.0 is released.
+  pyimgui = python3.pkgs.buildPythonPackage rec {
+    name = "pyimgui";
+    version = "1f095af5886f424ee12f26fa93b108b6420fafa4"; # 2.0 dev
+
+    src = fetchFromGitHub {
+      owner = "pyimgui";
+      repo = "pyimgui";
+      rev = version;
+      fetchSubmodules = true;
```

Comment:
Why not use our imgui?

Reply:
pyimgui uses cython to build imgui and generate bindings, it would require more than just replacing the path to make it use the nixpkgs package.